eng/Versions.props

```diff
@@ -4,13 +4,13 @@
     <MinorVersion>0</MinorVersion>
     <PatchVersion>40</PatchVersion>
     <SdkBandVersion>9.0.100</SdkBandVersion>
-    <PreReleaseVersionLabel>ci.main</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>servicing</PreReleaseVersionLabel>
```

Comment:
I think this has been changed for testing purposes, but it shouldn't be part of the changes, right?

Reply:
I think I made a mistake when creating the branch. 
I’ve resolved it manually.